LICENSE.md

```diff
@@ -1,11 +1,15 @@
-                    GNU GENERAL PUBLIC LICENSE
-                       Version 3, 29 June 2007
```

Comment:
Fix https://github.com/coala/git-task-list/issues/41 first

Reply:
Now https://github.com/coala/git-task-list/issues/46

Reply:
And now fix https://github.com/coala/git-task-list/issues/46

You should never modify a LICENSE file, not even to change the syntax.
It must be a verbatim copy of the license from a verifiable source.

Reply:
I miss understood with extra commits. @blazeu explained